modules/gapi/CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ set(gapi_srcs
     src/api/ginfer.cpp
     src/api/media.cpp
     src/api/rmat.cpp
+    src/api/gmeta_check.cpp
```

Comment:
a separate file may be too much for this simple function, consider `gproto.cpp` & its `_priv.hpp`.

Reply:
yeah, i thought about `_priv.hpp`, but I was confused with comment `// FIXME: why it is here?`

https://github.com/opencv/opencv/pull/20184/files#diff-dfc809caba75402a8e78080f173d484de3b67c66af4b57c81c680acf8275b6e4L22

And i realized that this include should be removed is in future
@dmatveev What do you think about that - do you still suggest to move it in `_priv.hpp` ?